test/index.js

```diff
@@ -1,5 +1,10 @@
 'use strict';
 
+process.config.target_defaults.default_configuration =
+  require('fs')
+    .readdirSync(require('path').join(__dirname, 'build'))
+    .filter((item) => (item === 'Debug' || item === 'Release'))[0];
```

Comment:
I don't know if you added the parens because you were worried about how arrow functions are parsed but they aren't necessary.  `=>` has lower precedence than `||`, it sits between assignment and the ternary operator.

Reply:
My worry was part of the reason, and it's good to know it was unwarranted. The other part was that the arrow function's two pieces are nicely grouped this way: `...) => (...` gives a nice separation, and makes it look less like `filter()` is taking more arguments but which are oddly not separated by commas.